docs/element-definitions.md

```diff
@@ -18,30 +18,23 @@ The Base API Element contains four properties: `element`, `meta`, `attributes`,
 
     The `element` property defines the name of element. It MUST be a string that references an element, which SHOULD be defined.
 
-- `meta` (enum)
+- `meta`
 
-    The `meta` property is a reserved object for Refract-specific values. When `meta` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
+    The `meta` property is a reserved object for Refract-specific values that MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
 
-    - Members
-        - (object)
-            - `id` - Unique Identifier, MUST be unique throughout the document
-            - `ref` (Element Pointer) - Pointer to referenced element or type
-            - `classes` (array[string]) - Array of classifications for given element
-            - `title` (string) - Human-readable title of element
-            - `description` (string) - Human-readable description of element
-            - `links` (array[Link Element]) - Meta links for a given element
-        - (array[Member Element])
+    - `id` ([String Element][]) - Unique Identifier, MUST be unique throughout the document
+    - `ref` ([Ref Element][]) - Pointer to referenced element or type
+    - `classes` (Array Element[[String Element][]]) - Array of classifications for given element
+    - `title` ([String Element][]) - Human-readable title of element
+    - `description` ([String Element][]) - Human-readable description of element
+    - `links` (Array Element[[Link Element][]]) - Meta links for a given element
 
-- `attributes` (enum)
+- `attributes`
 
     The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. When `attributes` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `attributes` and its properties and values.
 
     The `attributes` are used later in this document for representing data structures.
```

Comment:
Attributes should be an `object` with keys of string and values of Element. We should add this into spec.